hosted-cluster-config-operator/api/scheme.go

```diff
@@ -0,0 +1,59 @@
+package api
+
+import (
+	configv1 "github.com/openshift/api/config/v1"
+	oauthv1 "github.com/openshift/api/oauth/v1"
+	openshiftcpv1 "github.com/openshift/api/openshiftcontrolplane/v1"
+	operatorv1 "github.com/openshift/api/operator/v1"
+	"github.com/openshift/api/operator/v1alpha1"
+	osinv1 "github.com/openshift/api/osin/v1"
+	routev1 "github.com/openshift/api/route/v1"
+	securityv1 "github.com/openshift/api/security/v1"
+	hyperv1 "github.com/openshift/hypershift/api/v1alpha1"
+	prometheusoperatorv1 "github.com/prometheus-operator/prometheus-operator/pkg/apis/monitoring/v1"
+	corev1 "k8s.io/api/core/v1"
+	rbacv1 "k8s.io/api/rbac/v1"
+	apiextensionsv1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/serializer/json"
+	kasv1beta1 "k8s.io/apiserver/pkg/apis/apiserver/v1beta1"
+	auditv1 "k8s.io/apiserver/pkg/apis/audit/v1beta1"
+	apiserverconfigv1 "k8s.io/apiserver/pkg/apis/config/v1"
+	clientgoscheme "k8s.io/client-go/kubernetes/scheme"
+	apiregistrationv1 "k8s.io/kube-aggregator/pkg/apis/apiregistration/v1"
+)
+
+var (
+	Scheme = runtime.NewScheme()
+	// TODO: Even though an object typer is specified here, serialized objects
+	// are not always getting their TypeMeta set unless explicitly initialized
+	// on the variable declarations.
+	// Investigate https://github.com/kubernetes/cli-runtime/blob/master/pkg/printers/typesetter.go
+	// as a possible solution.
+	// See also: https://github.com/openshift/hive/blob/master/contrib/pkg/createcluster/create.go#L937-L954
+	YamlSerializer = json.NewSerializerWithOptions(
+		json.DefaultMetaFactory, Scheme, Scheme,
+		json.SerializerOptions{Yaml: true, Pretty: true, Strict: true},
```

Comment:
lol, a json serializer with `yaml: true`

Reply:
🤷‍♂️ it's what we use elsewhere